src/main.cpp

```diff
@@ -231,10 +231,13 @@ int main(int argc, char* argv[])
     // Non-GUI actions --help and --version as suggested by GNU coding standards,
     // section 4.7: http://www.gnu.org/prep/standards/standards.html#Command_002dLine-Interfaces
     QStringList texts;
-    if (!parsedCommandLineOk || parser.isSet(showHelp)) {
-        if (!parsedCommandLineOk) {
-            texts << append2LF.arg(QCoreApplication::translate("main", "Error: %1").arg(parser.errorText()));
-        }
+
+    if (!parsedCommandLineOk) {
+        // Warn of unknown options but tolerate them, in case user is not typing in command line.
+        std::cout << QCoreApplication::translate("main", "Error: %1\n").arg(parser.errorText()).toStdString();
```

Comment:
```suggestion
        std::cout << QCoreApplication::translate("main", "Warning: %1\n").arg(parser.errorText()).toStdString();
```

I would say it's a warning because we know squirrel options will get passed in (as SlySven enumerated them in https://github.com/Mudlet/Mudlet/pull/6583) but we don't have a need for them.

Reply:
It is pretty academic in the case for the Squirrel.windows installer as this message would never be visible to the end-user - even if they ran the `mudlet-*-installer.exe` from a Windows OS command line...